show/muxcable.py

```diff
@@ -413,3 +433,214 @@ def cableinfo(port):
 
     body = [[vendor, part_num]]
     click.echo(tabulate(body, headers=headers))
+
+
+@muxcable.group(cls=clicommon.AbbreviationGroup)
+def hwmode():
+    """Shows the muxcable hardware information directly"""
+    pass
+
+
+@hwmode.command()
+@click.argument('port', metavar='<port_name>', required=False, default=None)
+def muxdirection(port):
+    """Shows the current direction of the muxcable {active/standy}"""
+
+    per_npu_statedb = {}
+    transceiver_table_keys = {}
+    transceiver_dict = {}
+
+    # Getting all front asic namespace and correspding config and state DB connector
+
+    namespaces = multi_asic.get_front_end_namespaces()
+    for namespace in namespaces:
+        asic_id = multi_asic.get_asic_index_from_namespace(namespace)
+        per_npu_statedb[asic_id] = SonicV2Connector(use_unix_socket_path=False, namespace=namespace)
```

Comment:
The unit tests are failing b/c this part is different b/w the master and 202012 branches
```suggestion
        per_npu_statedb[asic_id] = swsscommon.SonicV2Connector(use_unix_socket_path=False, namespace=namespace)
```

Reply:
fixed

Reply:
@daall there are still some config_breakout failures.

Reply:
they are unrelated, it seems like there is some general issue with the check-build setup for 202012 in utilities. I can open a dummy PR later to debug this further, this change is OK